src/styles/header.scss

```diff
@@ -0,0 +1,113 @@
+header {
+    position: fixed;
+    z-index: 999;
```

Comment:
it's a bad practice to use z-index. 99% of the issue with layering of the components can be solved by propper composition